dicom-archive/DICOM/Element.pm

```diff
@@ -156,6 +237,27 @@ sub readFloat {
     return sprintf("%e", $val);
 }
 
+
+=pod
+
+=head3 readSequence($IN, $len)
+
+Reads Sequence.
+
```

Comment:
Skips over either a fixed number of bytes or over multiple sets of byte sequences delimited with specific byte values. When doing the latter, byte 0x00000000 is used to signal the end of the set of sequences. The sequence of bytes read is always discarded.

Reply:
done